sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_sender_async.py

```diff
@@ -191,8 +191,8 @@ async def send_transfer(self, message, *, send_async=False, **kwargs):
     async def cancel_transfer(self, delivery):
         try:
             index = self._pending_deliveries.index(delivery)
-        except ValueError:
-            raise ValueError("Found no matching pending transfer.")
+        except ValueError as exc:
```

Comment:
ValueError -> from None?

Reply:
I think this would just look like `ValueError: 10 is not in list`, so agree that it should be from None